package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "compile": "tsc index.ts test/spec.ts --module commonjs --noImplicitAny",
     "coverage": "cat ./coverage/lcov.info | coveralls",
-    "test": "istanbul cover _mocha",
+    "test": "istanbul cover node_modules/mocha/bin/_mocha",
```

Comment:
Yeah, this seems better. I also have a preference for `--reporter` over `-R` so it is clearer to other who may read it in the future.